### PR TITLE
Fix senior lesson.

### PR DESCRIPTION
**Pull request by khirayama**

### CODE REVIEW

File: src/senior/SeniorModel.coffee [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
+    @_events['change_data'] = @_events['change_data'] or []
+    @_events['change_data'].push(callback)
+
+  onChangeIsShowModal: (callback) ->
+    @_events['change_isShowModal'] = @_events['change_isShowModal'] or []
+    @_events['change_isShowModal'].push(callback)
+
+  addData: (item) ->
+    @_data.push(item)
+    @setData(@_data)
```
Comment (kjirou) [inline src/senior/SeniorModel.coffee]: `@setData(@_data)` is not necessary?

File: src/senior/SeniorModel.coffee [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
-      return
-    for callback in @_events[key]
-      (callback)()
-
-  onChange: (key, callback) ->
-    @_events[key] = @_events[key] || []
-    @_events[key].push(callback)
+  constructor: ->
+    @_events = {}
+    @_isShowModal = false
```
Comment (kjirou) [inline src/senior/SeniorModel.coffee]: Probably, `isShowModal` is invalid as English.
For example, `isModalShowing` is better than it, by the [GitHub code search](https://github.com/search)!

**Comment by kjirou** [comment]
@khirayama  I have completed the review. Please, confirm comments.

**Comment by khirayama** [comment]
I think it's all correct :+1:

**Comment by khirayama** [comment]
Fix them!

**Comment by kjirou** [comment]
[LGTM](http://lgtm.in/i/MehYxbPpq)

**Comment by khirayama** [comment]
merged!
I guess you can merge this repository. you are contributor.

**Comment by kjirou** [comment]
> I guess you can merge this repository. you are contributor.

All right.
Next it will do so.

**Pull request merged**